tests/expected_report.txt

```diff
@@ -8,14 +8,14 @@ tests/input/assignment-operators.php                  4       0
 tests/input/class-references.php                      10      0
 tests/input/concatenation_spacing.php                 24      0
 tests/input/constants-no-lsb.php                      2       0
-tests/input/constants-var.php                         3       0
+tests/input/constants-var.php                         4       0
```

Comment:
This change is because of https://github.com/slevomat/coding-standard/issues/771